examples/connectivity/plot_mne_inverse_label_connectivity.py

```diff
@@ -106,7 +107,7 @@
     label_ypos.append(ypos)
 
 # Reorder the labels based on their location
-lh_labels = [label for (ypos, label) in sorted(zip(label_ypos, lh_labels))]
+lh_labels = [label for (yp, label) in sorted(zip(label_ypos, lh_labels))]
```

Comment:
`ypos` is used later (or before?) so the variable name needed to be changed to avoid conflict